app/models/lunch.rb

```diff
@@ -60,9 +68,26 @@ def chalmrest
           summary, price = entry.summary.split('@')
           summary = summary.strip
           price = price.strip
-
+          if restaurant[:name] == "Express" || restaurant[:name] == "Kårrestaurangen"
```

Comment:
Same here, set it to `nil` instead.
Represent `no data` with `nil` and `no allergens` with `[]`